### PR TITLE
🔥(cli) remove click_log

### PR DESCRIPTION
**Pull request by SergioSim**
### Purpose

Ralph has a much more sophisticated logger setup and click_log
stopped functioning.

### Proposal

Remove click_log package dependency.

### CODE REVIEW

File: src/ralph/cli.py [hunk 1]
```diff
@@ -40,7 +37,6 @@
 
 
 @click.group(name="ralph")
-@click_log.simple_verbosity_option(get_root_logger())
 def cli():
     """Ralph is a stream-based tool to play with your logs"""
```
Comment (SergioSim) [inline src/ralph/cli.py:42]: I was wondering why passing verbosity options to ralph stopped having any effect on the logging.

Here `click_log.simple_verbosity_option` sets up the default logging configuration according to the verbosity options passed in the command line.

Later (line 43) `configure_logging` overwrites all the logging configuration according to environment variables / configuration file.

**Review by jmaupetit — COMMENTED** [review]
Good catch! As a substitute, can you please implement the two `-v` (verbose) and `-d` (debug) boolean flags as base CLI options that configure the root logger? :pray:

**Comment by SergioSim** [comment]
> Good catch! As a substitute, can you please implement the two `-v` (verbose) and `-d` (debug) boolean flags as base CLI options that configure the root logger? pray

Thanks) Good Idea! Updated the pull request to keep the verbosity option as it was before (but without click_log).

File: src/ralph/cli.py [hunk 2]
```diff
@@ -40,11 +38,20 @@
...
+    type=click.Choice(["CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"]),
+    metavar="LVL",
+    required=False,
+    help="Either CRITICAL, ERROR, WARNING, INFO or DEBUG",
+)
+def cli(verbosity=None):
     """Ralph is a stream-based tool to play with your logs"""
 
     configure_logging()
+    if verbosity:
```
Comment (jmaupetit) [inline src/ralph/cli.py:53]:
```suggestion
    if verbosity is not None:
```

File: tests/test_cli.py [hunk 3]
```diff
@@ -61,6 +61,22 @@ def test_extract_command_with_gelf_parser(gelf_logger):
...
+    result = runner.invoke(cli, ["-v", "CRITICAL", "extract", "-p", "gelf"], input=None)
+    assert result.output == ""
+    result = runner.invoke(cli, ["-v", "ERROR", "extract", "-p", "gelf"], input=None)
+    assert result.output == ""
+    result = runner.invoke(cli, ["-v", "WARNING", "extract", "-p", "gelf"], input=None)
+    assert result.output == ""
+    result = runner.invoke(cli, ["-v", "INFO", "extract", "-p", "gelf"], input=None)
+    assert "Extracting events using the gelf parser" in result.output
+    result = runner.invoke(cli, ["-v", "DEBUG", "extract", "-p", "gelf"], input=None)
+    assert "Extracting events using the gelf parser" in result.output
```
Comment (jmaupetit) [inline tests/test_cli.py:77]: Should be better to have an extra message to test the debug level?

**Review by jmaupetit — COMMENTED** [review]
Yay, thanks!

File: tests/test_cli.py (hunk 3 above)
Comment (SergioSim) [reply tests/test_cli.py:77]: Ok, I will give it a try)

Comment (SergioSim) [reply tests/test_cli.py]: Updated

**Comment by SergioSim** [comment]
Updated the pull request. Ready for a review ?

**Review by jmaupetit — COMMENTED** [review]
:tada: One last thing: can you lower case your commit title? :grimacing: :pray:

**Comment by jmaupetit** [comment]
And fill the CHANGELOG!

**Comment by SergioSim** [comment]
> tada One last thing: can you lower case your commit title? grimacing pray

Good catch! Thanks) Updated

File: CHANGELOG.md [hunk 4]
```diff
@@ -21,6 +21,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Support for application's configuration file
 - Add optional sentry integration
 - Distribute Arnold's tray to deploy Ralph in a k8s cluster as cronjobs
+- Remove click_log package dependency
```
Comment (jmaupetit) [inline CHANGELOG.md:24]: It should stand in the "Unreleased/Changed" sub-section (you should create it).

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,6 +8,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+- Remove click_log package dependency
```
Comment (SergioSim) [inline CHANGELOG.md:11]: > It should stand in the "Unreleased/Changed" sub-section (you should create it).

Wasn't aware of this. Thanks) Is it this section ?

Comment (jmaupetit) [inline CHANGELOG.md:11]:
```suggestion

### Changed

- Remove click_log package dependency
```

Comment (SergioSim) [reply CHANGELOG.md:11]: Ok, got it, Thanks)

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**